grammar.js

```diff
@@ -463,9 +462,12 @@ module.exports = grammar({
       $.set_comprehension,
       $.tuple,
       $.generator_expression,
-      $.ellipsis
+      $.ellipsis,
+      $.empty_expression
     ),
 
+    empty_expression: $ => seq('(',')'),
```

Comment:
In python, `()` is an empty tuple, not some other type of value. I'm verifying this by the fact that it participates in tuple concatenation:

```python
>>> (1, 2) + (3, 4)
(1, 2, 3, 4)

>>> () + (1, 2)
(1, 2)

>>> () + 1
TypeError: can only concatenate tuple (not "int") to tuple
```